### ISSUE TITLE
Test for WCAG success criterion 1.2.4: Captions (live)

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#204]
http://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-real-time-captions.html
The intent of this Success Criterion is to enable people who are deaf or hard of hearing to watch real-time presentations. Captions provide the part of the content available via the audio track. Captions not only include dialogue, but also identify who is speaking and notate sound effects and other significant audio.

This success criterion was intended to apply to broadcast of synchronized media and is not intended to require that two-way multimedia calls between two or more individuals through web apps must be captioned regardless of the needs of users. Responsibility for providing captions would fall to the content providers (the callers) or the “host” caller, and not the application.
## Applicable

live audio content in synchronized media
## Selector
- Video content
- Block where video can appear if live broadcasting is not available at the moment
## Failures
- None
## Techniques
- [ ] G9 (#220): Creating captions for live synchronized media 

AND
- [ ] G93: Providing open (always visible) captions
- [ ] G87 (#93): Providing closed captions using any readily available media format that has a video player that supports closed captioning 
## Relationship
- The technique of G9 is a consequence of G93 or G87: If G93 or G87 is true, G9 is automatically true.
- Probably can use G9 to do the detection of a livestream and return a not applicable or a can't tell. 

There are three situations: 
- a persistent livestream with continual events (broadcasting)
- a persistent livestream with incidental events
- an incidental livestream during an event
- It's hard to detect if this techniqu pass, however, we can do a 'cantTell' if a livestream is (sometimes) available, and a 'not applicable' if not.
